### PR TITLE
Fix version bump tests 

### PR DESCRIPTION
**Pull request by icristescu**

### CODE REVIEW

File: test/irmin-pack/test_pack_version_bump.ml [hunk 1]
```diff
@@ -121,56 +127,38 @@ end
...
   Lwt.return ()
 
 let tests =
   let f g () = Lwt_main.run @@ g () in
   Alcotest.
     [
-      test_case "test_RO_no_version_bump" `Quick (f test_RO_no_version_bump);
-      test_case "test_RW_no_version_bump" `Quick (f test_RW_no_version_bump);
-      test_case "test_RW_version_bump" `Quick (f test_RW_version_bump);
+      test_case "test_RO_no_version_bump" `Quick (f test_RO_no_migration);
```
Comment (tomjridge) [inline test/irmin-pack/test_pack_version_bump.ml:162]: test_case "test_RO_no_migration"?

File: test/irmin-pack/test_pack_version_bump.ml [hunk 2]
```diff
@@ -121,56 +127,38 @@ end
...
+      test_case "test_RW_no_version_bump" `Quick (f test_open_RW);
```
Comment (tomjridge) [inline test/irmin-pack/test_pack_version_bump.ml:163]: test_case "test_open_RW" ? (The existing text says "no version bump" which isn't correct I guess?)

**Comment by tomjridge** [comment]
Looks good to me.

File: test/irmin-pack/test_pack_version_bump.ml (hunk 2 above)
Comment (icristescu) [reply test/irmin-pack/test_pack_version_bump.ml]: thanks for spotting this

**Pull request merged**